src/tests/unit/tests/common/components/cards/instance-details.test.tsx

```diff
@@ -58,10 +58,10 @@ describe('InstanceDetails', () => {
             .verifiable(Times.once());
 
         const wrapper = shallow(<InstanceDetails {...props} />);
-        const tableElem = wrapper.find('table');
-        expect(tableElem.length).toBe(1);
+        const divElem = wrapper.find('.instance-details-card');
+        expect(divElem.length).toBe(1);
```

Comment:
not sure if this is needed or if it will make the error message better... but, if `divElem.length` is not 1, then the next line (`divElem.simulate('click');`) will failed with error message around 1 element was expected but found X.

Reply:
This is purely an extra assertion and can be removed. But I kept it to know that only one div with that class exists so as to avoid having multiple divs with the border decorations on UI. 
I would like to keep it until you strongly feel that I should remove it.